azure-mgmt-keyvault/src/main/java/com/microsoft/azure/management/keyvault/VaultAccessPolicyParameters.java

```diff
@@ -0,0 +1,13 @@
+package com.microsoft.azure.management.keyvault;
+import com.microsoft.azure.management.apigeneration.Fluent;
+import com.microsoft.azure.management.keyvault.implementation.VaultAccessPolicyParametersInner;
+import com.microsoft.azure.management.resources.fluentcore.model.HasInner;
+
+/**
+ * Parameters for updating the access policy in a vault.
+ */
+@Fluent(ContainerName = "/Microsoft.Azure.Management.Fluent.KeyVault")
+public interface VaultAccessPolicyParameters extends
+    HasInner<VaultAccessPolicyParametersInner>{
+

```

Comment:
@tiffanyachen similar comments I made about the DeletedVault interface; we should "promote" the properties from VaultAccessPolicyPropertiesInner class as methods of this interface. Also the interface should extend HasId(), HasName() and expose type() and location().